test/tests/services/resources-creator.test.js

```diff
@@ -70,6 +94,14 @@ describe('service > resources-creator', () => {
     const result = await new ResourcesCreator(IslandModel, { _id: '56cb91bdc3464f14678934ca', name: 'Haiti' })
       .perform();
     expect(result).toHaveProperty('name', 'Haiti');
-    expect(result.id).not.toBe('56cb91bdc3464f14678934ca');
+    expect(result._id).not.toBe('56cb91bdc3464f14678934ca');
```

Comment:
Silly question: does it mean that the test was "broken" (or at least didn't test the right thing) ?

Reply:
@arnaudbesnier the test was not testing the right value. `result.id` was always `undefined`, that's why it passed.